templates/controllers/tab/settings/productionStage/form/productionStageForm.tpl

```diff
@@ -46,4 +44,3 @@
 		{fbvFormButtons id="productionStageFormSubmit" submitText="common.save" hideCancel=true}
 	{/if}
 </form>
-<p><span class="formRequired">{translate key="common.requiredField"}</span></p>
```

Comment:
Good catch. I've pushed a new commit to remove those lingering required settings.